lib/fluent/plugin/in_tail.rb

```diff
@@ -120,7 +128,24 @@ def initialize
       config_argument :usage, :string, default: 'in_tail_parser'
     end
 
-    attr_reader :paths
+    config_section :group, param_name: :group, required: false, multi: false do
+      desc 'Regex for extracting group\'s metadata'
+      config_param :pattern, 
+                   :regexp,
+                   default: /^\/var\/log\/containers\/(?<podname>[a-z0-9]([-a-z0-9]*[a-z0-9])?(\/[a-z0-9]([-a-z0-9]*[a-z0-9])?)*)_(?<namespace>[^_]+)_(?<container>.+)-(?<docker_id>[a-z0-9]{64})\.log$/
+
+      desc 'Period of time in which the group_line_limit is applied'
+      config_param :rate_period, :time, default: 5
+
+      config_section :rule, param_name: :rule, required: true, multi: true do
+        desc 'Key-value pairs for grouping'
+        config_param :match, :hash, value_type: :regexp, default: {namespace: [DEFAULT_KEY], appname: [DEFAULT_KEY]}
```

Comment:
It seems the default value should be `{namespace: [DEFAULT_KEY], podname: [DEFAULT_KEY]}`.
```diff
- {namespace: [DEFAULT_KEY], appname: [DEFAULT_KEY]}
+ {namespace: [DEFAULT_KEY], podname: [DEFAULT_KEY]}
```

Reply:
Ah, thanks for pointing this out! I will fix this typo.